src/imageio/imageio.c

```diff
@@ -1493,7 +1493,9 @@ gboolean dt_imageio_export_with_flags(const dt_imgid_t imgid,
   dt_dev_cleanup(&dev);
   dt_mipmap_cache_release(&buf);
 
-  if(!thumbnail_export && strcmp(format->mime(format_params), "memory")
+  if(dt_lua_running()
```

Comment:
It looks like this will skip raising the signal if Lua is not compiled in, as well as not running.  Was that intended and correct behavior?

Reply:
Indeed the signal raising can/should be done depending on dt_control_running() ... Fixed with latest force-push